traffic_control/clients/python/setup.py

```diff
@@ -72,7 +72,7 @@
 	       packages=find_packages(exclude=["contrib", "docs", "tests"]),
 	       install_requires=[
 	           "future>=0.16.0",
-	           "requests>=2.13.0",
+	           "requests>=2.26.0",
```

Comment:
Should this be `requests[charset_normalizer]` instead of `requests`?

Reply:
No, this newer version of requests will automatically install charset_normalizer and use that if chardet isn't already installed (which it wouldn't be on fresh installs).